content/en/docs/reference/config/analysis/ist0103/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: PodMissingProxy
+layout: analysis-message
+---
+
+This message occurs when the sidecar is not present or working correctly.
+
+This most commonly occurs when you enable auto-injection but do not restart your
+pods afterwards, causing the sidecar to be missing.
+
+To resolve this problem, restart your pods and try again.
```

Comment:
Add sample command-line for how to do this.

Reply:
fixed